### PR TITLE
Move url to the registry

### PR DESCRIPTION
**Pull request by trisch-me**
## Changes

Moving url to the new registry. No new code, only refactoring of the existing structure. 

## Merge requirement checklist

* [x] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [ ] [CHANGELOG.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CHANGELOG.md) updated for non-trivial changes.
* [ ] [schema-next.yaml](https://github.com/open-telemetry/semantic-conventions/blob/main/schema-next.yaml) updated with changes to existing conventions.

### CODE REVIEW

**Review by ChrsMark — APPROVED** [review]
lgtm

**Review by pyohannes — APPROVED** [review]

File: docs/url/url.md [hunk 1]
```diff
@@ -25,11 +25,11 @@ This document defines semantic conventions that describe URL and its components.
 <!-- semconv url -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
```
Comment (joaopgrassi) [inline docs/url/url.md:26]: I wonder now with the registry, if we even need this page? Seems to be just duplicating things?

**Review by joaopgrassi — COMMENTED** [review]
I guess the paths changes all come from re-running the semconv tool, right?

**Comment by trisch-me** [comment]
yes, path changes are coming from running `make table-generation`

File: docs/url/url.md (hunk 1 above)
Comment (trisch-me) [reply docs/url/url.md:26]: Yes, particularly for this small attribute it's seems to be identical, but not for all attributes. Also I would need to update registry `url.md` with sensitive information from this file
This means that this file will be also removed from https://opentelemetry.io/docs/specs/semconv/ page, is it ok?

Comment (joaopgrassi) [reply docs/url/url.md:26]: Yeah. Let's leave it for now, we can revisit them later, once the registry is complete.

**Review by joaopgrassi — APPROVED** [review]

**Pull request merged**